Source/Holodeck/Tasks/Private/LocationTask.cpp

```
@@ -0,0 +1,69 @@
+#include "Holodeck.h"
+#include "LocationTask.h"
+#include "Json.h"
+
+// Set default values
+void ULocationTask::InitializeSensor() {
+	Super::InitializeSensor();
+}
+
+// Allows sensor parameters to be set programmatically from client.
+void ULocationTask::ParseSensorParms(FString ParmsJson) {
+	Super::ParseSensorParms(ParmsJson);
+
+	TSharedPtr<FJsonObject> JsonParsed;
+	TSharedRef<TJsonReader<TCHAR>> JsonReader = TJsonReaderFactory<TCHAR>::Create(ParmsJson);
+	if (FJsonSerializer::Deserialize(JsonReader, JsonParsed)) {
```

So if this doesn't work, it just silently fails?